app/assets/stylesheets/responsive/custom.scss

```diff
@@ -79,6 +79,12 @@ a:visited {
       color: $color_black;
     }
   }
+
+  details {
+    summary {
+      color: $color_pro_blue;
```

Comment:
You might also want to add `cursor: pointer;` here, as at least one browser I tested didn't do this by default